src/Appwrite/Platform/Modules/Databases/Http/Databases/Collections/Documents/XList.php

```
@@ -75,13 +75,13 @@ public function __construct()
             ->inject('response')
             ->inject('dbForProject')
             ->inject('user')
-            ->inject('queueForStatsUsage')
+            ->inject('usage')
             ->inject('transactionState')
             ->inject('authorization')
             ->callback($this->action(...));
     }
 
-    public function action(string $databaseId, string $collectionId, array $queries, ?string $transactionId, bool $includeTotal, int $ttl, UtopiaResponse $response, Database $dbForProject, Document $user, StatsUsage $queueForStatsUsage, TransactionState $transactionState, Authorization $authorization): void
+    public function action(string $databaseId, string $collectionId, array $queries, ?string $transactionId, bool $includeTotal, int $ttl, UtopiaResponse $response, Database $dbForProject, Document $user, Context $usage, TransactionState $transactionState, Authorization $authorization): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)